### PR TITLE
Initialize social media post (#241)

### PR DESCRIPTION
**Pull request by drmowinckels**

### CODE REVIEW

**Comment by drmowinckels** [comment]
@d4tagirl are you ok if we post this post next week? WE'd like to first to the posit conf summary post today, and then have the social media post come after 7 days (to give it a little time on the front page).

**Comment by d4tagirl** [comment]
Of course! In traveling so it’s even better for me!

File: content/news/2023-10-04-mastodon-meet-the-r-ladies/index.en.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+As R-Ladies Global’s commitment to building an inclusive and friendly community remains unwavering, we believe it is time to embrace the change that serves this commitment and enhances our community's experience. Following many of our community members, we are excited to share that we have joined Mastodon on the [Hachyderm server](https://hachyderm.io/about), a platform that we believe aligns more closely with our values. Mastodon provides us with an opportunity to create a space focused on meaningful interactions, inclusivity, and community building. We hope to make Mastodon an integral part of our communication strategy moving forward. If you are interested in learning more about how to interact on Mastodon, check out [this link](https://github.com/joyeusenoelle/GuideToMastodon).
```
Comment (shannonpileggi) [inline content/news/2023-10-04-mastodon-meet-the-r-ladies/index.en.md:21]:
```suggestion
As R-Ladies' commitment to building an inclusive and friendly community remains unwavering, we believe it is time to embrace the change that serves this commitment and enhances our community's experience. Following many of our community members, we are excited to share that we have joined Mastodon on the [Hachyderm server](https://hachyderm.io/about), a platform that we believe aligns more closely with our values. Mastodon provides us with an opportunity to create a space focused on meaningful interactions, inclusivity, and community building. We hope to make Mastodon an integral part of our communication strategy moving forward. If you are interested in learning more about how to interact on Mastodon, see this [guide to Mastodon](https://github.com/joyeusenoelle/GuideToMastodon).
```

**Review by shannonpileggi — APPROVED** [review]
A couple of small suggestions. Also, the post could use a preview image - any ideas?

**Comment by drmowinckels** [comment]
@shannonpileggi and @d4tagirl are we good to go? :D

**Review by shannonpileggi — APPROVED** [review]
looks good to me!

**Comment by shannonpileggi** [comment]
@drmowinckels when i look at the preview link i don't see the post but when i build the site locally i do. why is that?

**Comment by drmowinckels** [comment]
I had nested a build configure for drafts and future posts too much in the yaml. I think I got it right now

**Pull request merged**